config/owasp/suppressions.xml

```diff
@@ -31,4 +31,11 @@
     <cve>CVE-2016-5394</cve>
     <cve>CVE-2016-6798</cve>
   </suppress>
+  <suppress>
```

Comment:
could we please add a suppress until date depending when we think fix might be available ?

Reply:
added